contents/blog/why-redesign.mdx

```diff
@@ -0,0 +1,171 @@
+---
+title: Why are we redesigning PostHog?
+date: 2023-11-27
+author: ["cory-watilo"]
+rootpage: /blog
+featuredImage: ../images/blog/why-redesign/preview-3000.jpg
+featuredImageType: full
+category: PostHog news
+tags:
+  - Product updates
```

Comment:
I'll mirror some of this and point to it from changelog too. It should be a milestone.